otherfunctions.js

```diff
@@ -86,7 +86,7 @@ function compassNorth(){
 }
 
 function mapNavInfo(){
-    alert("On a TouchSceenUse 1 finger to Pan, 2 fingers to zoom and 3 fingers to rotate.\n Shortcut Keys: \n Ctrl + --> to rotate 15 degrees clockwise \n Ctrl + <-- to rotate 15 degrees anti clockwise");
+    alert("On a TouchScreen Use 1 finger to Pan, 2 fingers to zoom and 3 fingers to rotate.\n Shortcut Keys: \n Ctrl + --> to rotate 15 degrees clockwise \n Ctrl + <-- to rotate 15 degrees Anti clockwise");

```

Comment:
```suggestion
    alert("On a TouchScreen, Use 1 finger to Pan, 2 fingers to zoom and 3 fingers to rotate.\n Shortcut Keys: \n Ctrl + --> to rotate 15 degrees clockwise \n Ctrl + <-- to rotate 15 degrees Anti clockwise");
```

Reply:
Thank you for pointing this out. It seems that the changes I made affected the functionality of the rotate and next buttons in the navigator. Ill review the code surely..